pkg/devicetemplates/ttscsv.go

```diff
@@ -297,6 +314,28 @@ var csvFieldSetters = map[string]csvFieldSetterFunc{
 		}
 		return []string{"supports_class_c"}, nil
 	},
+	"vendor_id": func(dst *ttnpb.EndDevice, val string) ([]string, error) {
+		if dst.VersionIds == nil {
+			dst.VersionIds = &ttnpb.EndDeviceVersionIdentifiers{}
+		}
+		s, err := strconv.ParseUint(val, 10, 32)
```

Comment:
Is this field defined in our CSV reference as decimal?
<img width="745" alt="Screenshot 2022-08-03 at 14 10 47" src="https://user-images.githubusercontent.com/13186113/182604517-4001a09a-28d7-4721-82c1-ea436576cdd7.png">

In the TR005 spec, these values are hexadecimal.

Same for `vendor_profile_id ` below.